app/components/homepage-components/TestimonialCardButtons.tsx

```diff
@@ -0,0 +1,74 @@
+"use client";
+
+import Image from "next/image";
+import { useEffect, useState } from "react";
+import { useTheme } from "next-themes";
+import { motion } from "framer-motion";
+
+import {
+  arrowLeft,
+  arrowLeftDark,
+  arrowRight,
+  arrowRightDark,
+} from "@/public/svg-icons";
+
+interface ButtonsProps {
+  additionalClassesLeft: string;
+  additionalClassesRight: string;
+  previous: () => void;
+  next: () => void;
+}
```

Comment:
Even though the props are specifically related to this component, I still like the idea of moving them out into a separate folder to keep consistency within the application.

Reply:
All done